java/scene-layer-selection/README.md

```diff
@@ -0,0 +1,22 @@
+# Scene Layer Selection
+Select geoelements in a scene layer.
+
+![Scene Layer Select GeoElement App](scene-layer-selection.png)
```

Comment:
The image description should be similar to the module/image name. I think it should be **Scene Layer Selection App**